### ISSUE TITLE
Unsupported platform error

### ISSUE THREAD

**Issue by bjduarte** [diyor28/tf-docker-m1#2]
$ pip3 install https://github.com/diyor28/tf-docker-m1/releases/download/v1.0.0/tensorflow-2.8.0-cp39-cp39-linux_aarch64.whl 
ERROR: tensorflow-2.8.0-cp39-cp39-linux_aarch64.whl is not a supported wheel on this platform.

I have tried both available wheels and I get the same error when trying to execute either. I am using a MacBook Air (M1, 2020) running osX Monterey 12.3.1. I need to build docker containers to run tinsorflow and Jupiter from my Mac. Any help would be greatly appreciated.

**Comment by diyor28** [comment]
@bjduarte What python version do you have and what base image do you use for your docker container?

**Comment by bjduarte** [comment]
I have multiple versions of python but primarily I use 3.9.8. As for a base image for Docker I am not sure I know what you mean. Where would I insert a docker image? From the instructions on your Github I was suppose to install wheel, then run an install of your docker image. Where was I supposed to execute these commands, in an existing container, or from my host machine i.e. Macbook? Thanks for your reply. 

Bryan Duarte Ph.D.

**Comment by diyor28** [comment]
You are supposed to execute the from the inside of your docker container. Where did you execute them?

**Comment by bjduarte** [comment]
Ok so I am supposed to create a docker image then build a container, then execute this from the interactive shell of the container? Does it matter what image I use? As in can I use the python Ubuntu image then execute this? I need to run tinsorflow, jupyter, and python on my Macbook hardware. I was under the impression your github was for creating a docker image capable of running tinsorflow on Apple metal hardware. What does your dockerfile actually do? 

Bryan Duarte Ph.D.

**Comment by diyor28** [comment]
Yes, so my repo provides prebuilt .whl files for TensorFlow inside docker containers on Apple Silicon. The dockerfile included in the repo is for building TensorFlow from source. I suppose you don't want or need to do that. You can use pretty much any base image. I would personally recommend [official](https://hub.docker.com/_/python) python images. 
For example, you can do something like that:
```
FROM python:3.9 
pip install wheel
pip install  https://github.com/diyor28/tf-docker-m1/releases/download/v1.0.0/tensorflow-2.8.0-cp39-cp39-linux_aarch64.whl
CMD ["python"]
```
Afterwards run:
`docker build -t tensorflow .`
And you should have a docker container with TensorFlow installed running on your system. Or as you said yourself you can execute `pip install` from the interactive shell. If you want to install TensorFlow directly on your system without using docker, you can use https://pypi.org/project/tensorflow-macos/.
